Lib/test/test_fractions.py

```diff
@@ -1220,6 +1220,30 @@ def test_invalid_formats(self):
                 with self.assertRaises(ValueError):
                     format(fraction, spec)
 
+    @requires_IEEE_754
+    def test_float_format_testfile(self):
+        from test.test_float import format_testfile
+        with open(format_testfile, encoding="utf-8") as testfile:
+            for line in testfile:
+                if line.startswith('--'):
+                    continue
+                line = line.strip()
+                if not line:
+                    continue
```

Comment:
I suppose that such logic is used in other files using formatfloat_testcases.txt. Is there a good place to share common code?

If we add mathdata/, can we add a helper function there which would iterate on the file and strip comments?

Reply:
I was going to add it in `test_float`, but it is simple enough, so I just copied it.